drizzlepac/haputils/analyze.py

```diff
@@ -425,13 +426,30 @@ def analyze_data(input_file_list, log_level=logutil.logging.DEBUG, type=""):
         # Determine if the image has one of these conditions.  The routine
         # will exit processing upon the first satisfied condition.
 
+        # Check if all science image arrays are filled with zero values
```

Comment:
Change comment to be "...all science image arrays in the RAW file...".  SCI extensions in the processed data are float values which would require a different check against zero.